geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureBlobStoreInfo.java

```diff
@@ -0,0 +1,267 @@
+/**
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * <p>You should have received a copy of the GNU Lesser General Public License along with this
+ * program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Andrea Aime, GeoSolutions, Copyright 2019
+ */
+package org.geowebcache.azure;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.geowebcache.config.BlobStoreInfo;
+import org.geowebcache.layer.TileLayerDispatcher;
+import org.geowebcache.locks.LockProvider;
+import org.geowebcache.storage.BlobStore;
+import org.geowebcache.storage.StorageException;
+
+/** Plain old java object representing the configuration for an S3 blob store. */
```

Comment:
Docs reference S3 rather than Azure in several places

Reply:
Whoops, fixing.